src/NUnitFramework/framework/Constraints/NUnitEqualityComparer.cs

```diff
@@ -63,6 +63,11 @@ public sealed class NUnitEqualityComparer
         /// </summary>
         private bool _ignoreWhiteSpace;
 
+        /// <summary>
+        /// If true, all string comparisons will be done after normalizing newlines
+        /// </summary>
+        private bool _NormalizeLineEndings;
```

Comment:
I guess I'll have to fix this copypasta. I can do this after the first round of reviews.

```suggestion
        private bool _normalizeLineEndings;
```